mmdeploy/backend/vacc/backend_manager.py

```diff
@@ -0,0 +1,144 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import logging
+import sys
+from typing import Any, Callable, Optional, Sequence
+
+from mmdeploy.utils import get_common_config, get_model_inputs, get_root_logger
+from ..base import BACKEND_MANAGERS, BaseBackendManager
+
+
+@BACKEND_MANAGERS.register('vacc')
+class VACCManager(BaseBackendManager):
+
+    @classmethod
+    def build_wrapper(cls,
+                      backend_files: Sequence[str],
+                      device: str = 'cpu',
+                      input_names: Optional[Sequence[str]] = None,
+                      output_names: Optional[Sequence[str]] = None,
+                      deploy_cfg: Optional[Any] = None,
+                      **kwargs):
+        """Build the wrapper for the backend model.
+
+        Args:
+            backend_files (Sequence[str]): Backend files.
+            device (str, optional): The device info. Defaults to 'cpu'.
+            input_names (Optional[Sequence[str]], optional): input names.
+                Defaults to None.
+            output_names (Optional[Sequence[str]], optional): output names.
+                Defaults to None.
+            deploy_cfg (Optional[Any], optional): The deploy config. Defaults
+                to None.
+        """
+        from .wrapper import VACCWrapper
+
+        # For unittest deploy_config will not pass into _build_wrapper
+        # function.
+
+        try:
+            common_cfg = get_common_config(deploy_cfg)
+            vdsp_params_info = common_cfg['vdsp_params_info']
+
+            return VACCWrapper(
+                lib_file=backend_files[0],
+                graph_file=backend_files[1],
+                param_file=backend_files[2],
+                vdsp_params_info=vdsp_params_info,
+                output_names=output_names)
+        except Exception:
+            print('Build model process success, wrapper process stopped')
```

Comment:
I think it's `build model failed`, isn't it?

Reply:
no. it's `build model success`, but `wrapper failed` because of `vdsp_params_info` not setted or other